mesh/pmesh.hpp

```diff
@@ -349,6 +366,19 @@ class ParMesh : public Mesh
    /// Old mesh format (Netgen/Truegrid) version of 'PrintAsOne'
    void PrintAsOneXG(std::ostream &out = mfem::out);
 
+   /** Print the mesh in parallel PVTU format. The PVTU and VTU files will be
+       stored in the directory specified by @a pathname. If the directory does
+       not exist, it will be created. */
+   virtual void PrintVTU(std::string pathname,
+                         VTKFormat format=VTKFormat::ASCII,
+                         bool high_order_output=false,
+                         int compression_level=0,
+                         bool bdr=false);
```

Comment:
How does this play together with ParGridFunctions?

Reply:
I am not sure, hopefully this is orthogonal, I just rearranged the `Print*` declarations that seemed scattered in the header file.